newton/_src/viewer/viewer_viser.py

```
@@ -627,11 +670,54 @@ def end_frame(self):
+                            series=(
+                                uplot.Series(label="step"),
+                                uplot.Series(label=name, stroke="#3b82f6", width=2),
+                            ),
+                            scales={"x": uplot.Scale(time=False)},
+                            aspect=2.0,
+                        )
+                    self._plot_handles[name] = handle
+                else:
+                    handle.data = (x, y)
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)